src/System.IO.FileSystem/src/System.IO.FileSystem.csproj

```diff
@@ -192,12 +192,15 @@
     <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.DeleteVolumeMountPoint.cs">
       <Link>Common\Interop\Windows\Interop.DeleteVolumeMountPoint.cs</Link>
     </Compile>
+<<<<<<< HEAD
     <Compile Include="$(CommonPath)\System\Memory\FixedBufferExtensions.cs">
       <Link>Common\System\Memory\FixedBufferExtensions.cs</Link>
     </Compile>
   </ItemGroup>
   <!-- Windows : Win32 only -->
   <ItemGroup Condition="'$(TargetsWindows)' == 'true' and '$(UWPCompatible)' != 'true'">
```

Comment:
The ItemGroup moved down- you should be able to remove 199-201.

Reply:
Oops, didn't see this conflict. Will fix it.

Reply:
Fixed :)